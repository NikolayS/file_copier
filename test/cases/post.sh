#!/bin/bash

# to run this scenario, COPIERSERVICE should be defined (`export COPIERSERVICE=https://your.copier.host.name`)

echo '-----011000010111000001101001
Content-Disposition: form-data; name="src"

http://img0.liveinternet.ru/images/attach/c/9/106/906/106906542_3925311_jenshina_za_rylyom.jpg
-----011000010111000001101001
Content-Disposition: form-data; name="md5"

1
-----011000010111000001101001
Content-Disposition: form-data; name="wh"

1
-----011000010111000001101001--' |  \
      http POST "$COPIERSERVICE" \
        cache-control:no-cache \
          content-type:'multipart/form-data; boundary=---011000010111000001101001' \
            -h | grep -v "Date:"
