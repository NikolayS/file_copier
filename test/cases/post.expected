HTTP/1.1 200 OK
Transfer-Encoding: chunked
X-File-Copier-Md5: f87818ad1d6ebaddeb7649c620c861cc
X-File-Copier-Img-Width: 400
X-File-Copier-Img-Height: 400
X-File-Copier-Size: 44739
X-Location: /data/b6/6b/5a/21/b66b5a21c3bd5f160356dcf55bde619fc2265ce39dd0a333be786cd594cf330f.jpg
Access-Control-Expose-Headers: X-File-Copier-Size, X-Location, X-File-Copier-Img-Height, X-File-Copier-Img-Width, X-File-Copier-Md5

